web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -141,6 +127,55 @@ export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement =>
     );
   };
 
+  const emptyForYouMessage = (): ReactNode => {
+    const showEmptyOpportunitiesMsg =
+      props.displayCertifications &&
+      props.displayFundings &&
+      props.certifications.length + props.fundings.length === 0;
+
+    const showCompleteRequiredTasksMsg =
+      !props.displayCertifications &&
+      !props.displayFundings &&
+      props.topCards.length + props.bottomCards.length === 0;
+
+    const isRemoteSellerWorker = (): boolean => {
+      if (business) {
+        const foreignBusinessType = determineForeignBusinessType(business.profileData.foreignBusinessTypeIds);
+        return foreignBusinessType === "REMOTE_SELLER" || foreignBusinessType === "REMOTE_WORKER";
+      }
+      return false;
+    };
+
+    const showEmptyForYouMessage =
+      showEmptyOpportunitiesMsg || showCompleteRequiredTasksMsg || isRemoteSellerWorker();
+
+    const getTopText = (): string => {
+      if (isRemoteSellerWorker()) return Config.dashboardDefaults.emptyOpportunitiesRemoteSellerWorkerText;
```

Comment:
pebble - from a content/cms perspective, I think we may need go update the preview so it renders the component